backend/spellchecker/libparser/parser.cpp

```diff
@@ -0,0 +1,47 @@
+#include "parsers/latexparser.hxx"
+#include <cstring>
+#include <iostream>
+#include <assert.h>
+
+extern "C" {
+
+	struct Token {
+		char* token;
+	};
+
+	std::string alphabet("qwertzuiopasdfghjklyxcvbnmQWERTZUIOPASDFGHJKLYXCVBNM");
```

Comment:
Yeah, I know, but for some reason latexparser needs alphabet as an argument of constructor. textparser (latexparser inherits it) has default constructor, btw, and this is the very string that initializes textparser's wordchar array (some inner stuff, I dunno).
Proof (textparser.cxx:126):
![2016-09-01__00 38 57__592x74](https://cloud.githubusercontent.com/assets/4928316/18147304/b2a2ad96-6fdc-11e6-8bad-8e58a1eb993f.png)